include/pacemaker.h

```diff
@@ -26,6 +26,19 @@ extern "C" {
 
 #  include <crm/stonith-ng.h>
 
+/*!
+ * \brief Modify operation of running a cluster simulation.
+ */
+typedef enum {
+    pcmk_sim_none             = 0,
+    pcmk_sim_all_actions      = 1 << 0,
+    pcmk_sim_show_pending     = 1 << 1,
+    pcmk_sim_process          = 1 << 2,
+    pcmk_sim_show_scores      = 1 << 3,
+    pcmk_sim_show_utilization = 1 << 4,
+    pcmk_sim_simulate         = 1 << 5,
+} pcmk_sim_flags_e;
```

Comment:
If we want to standardize on typedef'ing enums, then the \_e name ending is a good idea; if we want to standardize on not typedef'ing, then we don't need \_e

Reply:
Fixed.